test/unit/snowflake_test.js

```diff
@@ -791,7 +791,7 @@ describe('connection.execute() statement successful', function ()
 
                 assert.strictEqual(statement.getNumRows(), 1);
                 assert.ok(Util.isObject(statement.getSessionState()));
-                assert.ok(Util.string.isNotNullOrEmpty(statement.getStatementId()));
+                assert.ok(Util.string.isNotNullOrEmpty(statement.getQueryId()));
```

Comment:
here also we should keep both assertions for backward compatibility tests

Reply:
I will add them

Reply:
how about this test?